packages/babel-plugin-jsx/src/transform-vue-jsx.ts

```diff
@@ -30,10 +36,10 @@ const transformJSXSpreadAttribute = (t, path, mergeArgs) => {
   }
 };
 
-const getJSXAttributeValue = (t, path, state) => {
-  const valuePath = path.get('value');
+const getJSXAttributeValue = (path: JSXAttriPath, state: State) => {
+  const valuePath = path.get<'value'>('value');
```

Comment:
`path.get('value')`

Reply:
Better use generics to infer the type of valuePath